tests/brevitas/hyp_helper.py

```diff
@@ -218,3 +220,21 @@ def min_max_tensor_random_shape_st(draw, min_dims=1, max_dims=4, max_size=3, wid
     min_tensor = torch.tensor(min_list).view(*shape)
     max_tensor = torch.tensor(max_list).view(*shape)
     return min_tensor, max_tensor
+
+
+@st.composite
+def random_minifloat_format(draw, min_bit_width=MIN_INT_BIT_WIDTH, max_bit_with=MAX_INT_BIT_WIDTH):
+    """"
+    Generate a minifloat format. Returns bit_width, exponent, mantissa, and signed.
```

Comment:
Add a `TODO` here. The minifloat format will be different once we have support for a FloatQuantTensor (Soon TM)